sdk/release/RELEASE.md

```diff
@@ -162,27 +154,93 @@ windows` inside the `daml-language-ad-hoc` project. This command prints IP
 1. Run `daml version --assistant=yes` and verify that the new version is
    selected as the assistant version and the default version for new projects.
 
-1. Run `daml new myproject` to create a new project and switch to it using
-   `cd myproject`.
+1. Run `daml new myproject --template multi-package-example` to create a new project and switch to its main package using
+   `cd myproject/main`.
 
 1. Run `daml start`. Wait for the yellow message confirming the sandbox is ready for use.
 
 1. Kill `daml start` with `Ctrl-C` (followed by "y" because Windows).
 
-1. Run `daml studio --replace=always` and open `daml/Main.daml`. Verify that
-   there is a `Script results` link on top of the script, click on it, and
+1. Build all packages by moving out of the main package (`cd ..`) and running `daml build --all`.
```

Comment:
I think you applied my suggested change from https://github.com/digital-asset/daml/pull/21990#discussion_r2432570455 to this place, but the second part got skipped:
> An indication of what you should see when running an all build would be useful - e.g. should see "Using multi-package.json" message